odoo/fields.py

```diff
@@ -933,7 +933,7 @@ def update_db_index(self, model, column):
             except psycopg2.OperationalError:
                 _schema.error("Unable to add index for %s", self)
         else:
-            sql.drop_index(model._cr, indexname, model._table)
+            _schema.info("Keep unexpected index %s on table %s", indexname, model._table)
```

Comment:
![Selection_183](https://user-images.githubusercontent.com/25005517/130463435-9eea6987-4f1f-4017-abb0-53f3dc2a70a0.png)
in our CI

Reply:
Yeah, sorry for that. We already have a PR for that: https://github.com/odoo/odoo/pull/75387